src/solver/ia_main.jl

```diff
@@ -0,0 +1,250 @@
+using Symbolics
+E = Base.MathConstants.e
```

Comment:
I think it is fine to use `Base.MathConstants.e` in this file and remove this line. 
Otherwise, this can probably be `const`.